src/ArmMatrixWrapper.h

```diff
@@ -210,13 +212,144 @@ class ArmMatrixWrapper {
         return mseValue;
     };
 
-    // // TEMPORARY!!!
-    // // Fast Fourier Transform Wrapper (FFT)
-    // // Assumes input is real data row vector, and output will be complex row vector
-    // ArmMatrixWrapper<1, MaxCols> fft() const {
-    //     ArmMatrixWrapper<1, MaxCols> result;
-    //     arm_cfft_f32(&arm_cfft_sR_f32_len, reinterpret_cast<float32_t*>(data), 0, 1);
-    //     memcpy(result.data, data, MaxBufferSize * sizeof(float32_t));
-    //     return result;
-    // };
+    // Computes the raw FFT for specific channel given a number of samples 
```

Comment:
Can we also add a pretty print function? :)
I was going to add it to my branch, but I think it makes sense in this PR.
Something like:
```
printk("\nmatrix data:\n")
for (row in matrix.numRows) {
  for (col in matrix.numCols) {
    printk("%.4f\t", this.at(row,col))
  }
}
printk("\n")
```